src/types/common.d.ts

```diff
@@ -19,12 +18,12 @@ export type TxError =
 
 export type NetworkName = "testnet" | "mainnet";
 
-export interface Asset extends TokenInfo {
-  address: string;
-  iconSrc?: string;
-  balance: string;
-  updatedAt?: Date | string | number;
-}
+// export interface Asset extends TokenInfo {
```

Comment:
better to remove this, it remains history so we can use it when needed in the future